libs/driver/in-memory/src/checkout/in-mem/checkout.testing.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, Inject } from '@angular/core';
+
+import { DaffCartFactory, DaffCartItemFactory, DaffProductImageFactory} from '@daffodil/core/testing';
+
+import { InMemoryDbService, STATUS } from 'angular-in-memory-web-api';
+import { DaffDriverInterface, DaffDriver } from '@daffodil/driver';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryCheckoutTestingService implements InMemoryDbService {
+  private order;
+
+  constructor(
+    private cartFactory: DaffCartFactory,
+    private cartItemFactory: DaffCartItemFactory,
+    private productImageFactory: DaffProductImageFactory,
+    @Inject(DaffDriver) public driver: DaffDriverInterface
+  ) {
+    this.order = this.cartFactory.create({ 
```

Comment:
Don't construct this here. We should be constructing it in the post method based on the content from `placeOrder`.

Reply:
removed